api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -306,6 +306,8 @@ message ExtAuthz {
   //
   // If this is false the filter will not emit stats, but filter_metadata will still be respected if
   // it has a value.
+  //
+  // Fields ``latency``, ``bytesSent`` and ``bytesReceived`` are exposed for CEL and logging
```

Comment:
The comment on L304 indicates if not using Envoy gRPC, only latency is emitted.  Does this comment override that one?  If so, would be good to update that comment too.

Reply:
Updated. Let me know if the new formulation is clearer.